src/main/java/net/neoforged/neoforge/fluids/FluidType.java

```diff
@@ -868,6 +879,7 @@ public static final class Properties {
         private Rarity rarity = Rarity.COMMON;
         @Nullable
         private DripstoneDripInfo dripInfo;
+        private boolean isWaterLike = true;
```

Comment:
This should default to false as otherwise we would be introducing a breaking change to other fluids that aren't supposed to be water like.

Reply:
Most of the properties we have for water stuff are true default so i matched that. But I'll reverse this one